org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/lsp/JavaProtocolExtensions.java

```diff
@@ -97,4 +99,10 @@
 
 	@JsonRequest
 	CompletableFuture<RefactorWorkspaceEdit> getRefactorEdit(GetRefactorEditParams params);
+
+	@JsonRequest
+	CompletableFuture<PackageDestinationsResponse> getPackageDestinations(String[] documentUris);
```

Comment:
I think `getMoveDestinations` is a better fit, as we might get other actions that need to return other package destinations, using different criteria